src/Foundatio/Caching/InMemoryCacheClient.cs

```diff
@@ -825,9 +849,9 @@ public async Task SetExpirationAsync(string key, TimeSpan expiresIn)
         }
 
         Interlocked.Increment(ref _writes);
-        if (_memory.TryGetValue(key, out var value))
+        if (_memory.TryGetValue(key, out var existingEntry))
```

Comment:
I renamed them as there was like 3 or 4 different names for existing cache entries and it was driving me crazy while debugging.